### PR TITLE
Fix unicode bug in web plugin

### PR DESCRIPTION
**Pull request by waweic**
Added Exception and unidecode to web plugin. Added Description to Changelog

### CODE REVIEW

**Comment by waweic** [comment]
Ok... Doesn't seem to work in Python 2.7, I will fix this later

**Comment by waweic** [comment]
Now fixed and tested. Seems to work on both Python 2.7 and Python 3

File: beetsplug/web/__init__.py [hunk 1]
```diff
@@ -225,10 +226,22 @@ def item_file(item_id):
...
 
+    try:
+        unicode_item_path = util.text_string(item.path)
+    except (UnicodeDecodeError, UnicodeEncodeError):
+        unicode_item_path = u"fallback" + os.path.splitext(item_path)[1]
+
+    try:
+        # Imitate http.server behaviour
+        os.path.basename(unicode_item_path).encode("latin-1", "strict")
+        safe_filename = os.path.basename(unicode_item_path)
```
Comment (sampsyo) [inline beetsplug/web/__init__.py]: This line should probably go in an `else` case for the try/catch block, which runs when an exception does not occur.

**Comment by sampsyo** [comment]
Looks perfect. Thank you!

**Pull request merged**